plugins/@grouparoo/google-sheets/src/lib/sheet-import/downloadAndRefreshFile.ts

```diff
@@ -52,6 +54,9 @@ export async function downloadAndRefreshFile(
       rows = await sheet.read({ limit, offset });
     }
     writer.end();
+
+    if (fs.existsSync(localPath)) fs.rmSync(localPath, { force: true });
```

Comment:
I wonder if this works without deleting first. then it's totally atomic.

Reply:
(this should be fine - the sync methods aren't async or callbacks, so the node process won't go elsewhere)